packages/api/api-spec.json

```diff
@@ -500,9 +500,6 @@
           "thumbnails": {
             "$ref": "#/components/schemas/ThumbnailsDto"
           },
-          "tier": {
-            "type": "number"
-          },
```

Comment:
I think we agreed we should return subscribers count in place of the tier?

Reply:
Addressed in [c2b9371](https://github.com/Joystream/youtube-synch/pull/63/commits/c2b9371103608cac667deae29f20e285398c90b6)